keyboards/kopibeng/typ65/info.json

```
@@ -9,7 +9,7 @@
         "device_version": "0.0.1"
     },
     "layouts": {
```

As this is targeting `master`, we should aim to maintain existing compatibility
```suggestion
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT"
    },
    "layouts": {
```